test/feature-benchmark/scenarios_subscribe.py

```diff
@@ -0,0 +1,134 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from random import getrandbits
+from textwrap import dedent
+
+from materialize.feature_benchmark.measurement_source import MeasurementSource, Td
+from materialize.feature_benchmark.scenario import Scenario
+
+
+class SubscribeParallel(Scenario):
+    """Feature benchmarks related to SUBSCRIBE"""
+
+    SCALE = 2  # So 100 concurrent SUBSCRIBEs by default
```

Comment:
More than 100 is too slow?

Reply:
yes, https://github.com/MaterializeInc/materialize/issues/18261 , will reference the issue in the test.